php_fpm-formula/php-fpm/macros.jinja

```diff
@@ -0,0 +1,26 @@
+{#-
+Jinja macros file for the PHP FPM Salt states
+Copyright (C) 2025 Georg Pfuetzenreuter <mail+opensuse@georg-pfuetzenreuter.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- set boolmap = {true: 'on', false: 'off'} -%}
+
+{%- macro value(v, pool) -%}
```

Comment:
The `pool` parameter is unused, and all callers only call the macro with one parameter.

Reply:
Removed. Interesting Jinja behavior that it does not fail as there's no default value.